docs/usage/cmd_deploy.rst

```diff
@@ -0,0 +1,121 @@
+==================================
+Deploying a new application config
+==================================
+
+``shpkpr deploy`` allows you to deploy a new (or changes to an existing) application configuration by rendering and POSTing a JSON template to Marathon::
+
+    $ shpkpr deploy --help
+    Usage: shpkpr deploy [OPTIONS]
+
+      Deploy application from template.
+
+    Options:
+      -t, --template FILENAME  Path of the template to use for deployment.
```

Comment:
We've got the `template` and `env_prefix` option here, should we add the `marathon_url` option here too?